examples/olives/js/lib/Tools.js

```diff
@@ -1,20 +1,17 @@
-/*global define*/
-(function () {
-	'use strict';
+'use strict';
 
-	/*
-	 * A set of commonly used functions.
-	 * They're useful for several UIs in the app.
-	 * They could also be reused in other projects
-	 */
-	define('Todos/Tools', {
-		// className is set to the 'this' dom node according to the value's truthiness
-		'toggleClass': function (value, className) {
-			if (value) {
-				this.classList.add(className);
-			} else {
-				this.classList.remove(className);
-			}
-		}
-	});
-})();
+/*
+ * A set of commonly used functions.
+ * They're useful for several UIs in the app.
+ * They could also be reused in other projects
+ */
+module.exports = {
+    // className is set to the 'this' dom node according to the value's truthiness
+    'toggleClass': function (value, className) {
+        if (value) {
+            this.classList.add(className);
+        } else {
+            this.classList.remove(className);
+        }
+    }
```

Comment:
For this file same thing as https://github.com/tastejs/todomvc/pull/1242#discussion_r27489362